src/app/containers/InlineLink/index.jsx

```diff
@@ -6,12 +6,15 @@ import InlineLink from '../../components/InlineLink';
 const InternalInlineLink = InlineLink.withComponent(Link);
 
 const InlineLinkContainer = ({ href, children, ...rest }) => {
-  const regex = '^/news/articles/c[a-zA-Z0-9]{10}o$';
+  const schemeHostPrefix = 'https://www.bbc.(co.uk|com)';
+  const regex = RegExp(`^${schemeHostPrefix}/news/articles/c[a-zA-Z0-9]{10}o$`);
```

Comment:
This will also need to match the various environments EG: 
`https://www(.int|.test|.stage|).bbc.(co.uk|com)`